examples/pymc3_howto/sampling_compound_step.ipynb

```diff
@@ -12,10 +12,24 @@
     "# Compound Steps in Sampling\n",
```

Comment:
<p>"What happens to <s>variables</s> that occur in multiple <s>samplers</s>? " it should be sample statistics and step methods, actual variables can't be sampled by two step methods at once. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/141/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>